include/votca/tools/structureparameters.h

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2009-2019 The VOTCA Development Team (http://www.votca.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#pragma once
+#ifndef VOTCA_TOOLS_STRUCTUREPARAMETERS_H
+#define VOTCA_TOOLS_STRUCTUREPARAMETERS_H
+
+#include "unitconverter.h"
+#include <Eigen/Dense>
+#include <boost/any.hpp>
+#include <cassert>
+#include <unordered_map>
+
+namespace votca {
+namespace tools {
+
```

Comment:
Can you explain this a bit more?

Reply:
So the comments I thought were quite explicit, if you can target which part you are a little confused about I can try to clear it up though.

Reply:
why structure parameters and not simply units?

Reply:
ignore that please. I still do not get the advantage of the structure parameter class

Reply:
Units is too general, an id is not really a unit any way (int would be the unit), these are meant to be parameters passed to a class that is related to the structure, molecules, fragments, motifs etc.